src/main/java/ch/njol/skript/aliases/ItemType.java

```diff
@@ -346,6 +346,13 @@ public boolean hasBlock() {
 		}
 		return false;
 	}
+
+	/**
+	 * @return Whether this ItemType has at least one ItemData that represents it
+	 */
+	public boolean hasType() {
```

Comment:
I don't think this method is needed, you should be using `isAll()` to determine if the ItemType will return multiple or single or the other method `hasItem()` which has the exact same JavaDoc description.

Reply:
This is actually different, it doesn't check for whether it's an item or a block, it just checks if it has any kind of ItemData (types)